src/showcase/app/business/business-examples/notification-toast-examples/notification-toast-component-example/notification-toast-component-example.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { SbbNotificationToast } from '@sbb-esta/angular-business/notification-toast';
+
+@Component({
+  selector: 'sbb-notification-toast-component-example',
+  templateUrl: './notification-toast-component-example.component.html',
+})
+export class NotificationToastComponentExampleComponent {
+  constructor(private _notification: SbbNotificationToast) {}
+
+  showNotification() {
+    this._notification.openFromComponent(ExampleToastComponent);
+  }
+}
+
+@Component({
+  selector: 'sbb-example-toast',
+  template: `<span class="example-style">SBB CFF FFS</span>`,
+  styles: [
+    `
+      .example-style {
+        color: darkred;
```

Comment:
for me, this color looks weird, maybe take an sbb color? oder just style another thing than color :)

Reply:
Changed to font style italic